db/migrate/20171028221157_add_reblogs_to_follows.rb

```diff
@@ -0,0 +1,21 @@
+require Rails.root.join('lib', 'mastodon', 'migration_helpers')
+
+class AddReblogsToFollows < ActiveRecord::Migration[5.1]
+  include Mastodon::MigrationHelpers
+
+  safety_assured do
+    disable_ddl_transaction!
+  end
+
+  def up
+    safety_assured do
+      add_column_with_default :follows, :show_reblogs, :boolean, default: true, allow_null: false
```

Comment:
Is there a way to at least make this `add_column` compatible with strong_migrations' requirements?  I'm not too concerned about the `follow_requests` table because `AfterRemoteFollowRequestWorker` will clean them up, but `follows` is much bigger.